### PR TITLE
[enterprise-4.12] OCPBUGS#13406: Update the ansible and helm base image version tags

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12 only

Issue: https://issues.redhat.com/browse/OCPBUGS-13406

Link to docs preview: https://60071--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/helm/osdk-helm-updating-projects.html#osdk-upgrading-projects_osdk-helm-updating-projects

https://60071--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/ansible/osdk-ansible-updating-projects.html#osdk-upgrading-projects_osdk-ansible-updating-projects

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/osdk-updating-v1220-to-v1250.adoc [hunk 1]
```diff
@@ -279,6 +289,14 @@ ifdef::ansible[]
   changed_when: false
 ----
 <1> Replace `--load_restrictor none .` with `--load-restrictor LoadRestrictionNone`.
+
+. Find the `ose-ansible-operator` pull spec in Dockerfile, and update the image tag to `v4.12`:
```
Comment (michaelryanpeter) [inline modules/osdk-updating-v1220-to-v1250.adoc:293]: If you follow my suggestion above, this section can be removed.

**Review by michaelryanpeter — COMMENTED** [review]
Thank you for working on this bug. I made a modular documentation suggestion that would make this easier to maintain in the future. 

Since I also write for the Operator SDK, this review is a bit more in depth than a typical peer review, but I hope that it helps. If you have questions, please reach out. I am happy to help explain my rationale.

Thank you!

**Comment by stevsmit** [comment]
LGTM

**Pull request merged**